### ISSUE TITLE
Searching for songs doesn't work?

### ISSUE THREAD

**Issue by chiarastrq** [UnderMybrella/EternalJukebox#135]
### Before
- [yes] I have checked for similar issues already
- [ ] My issue is not a bad syncronisation between the Spotify track and the YouTube audio (Those go [here.](https://github.com/UnderMybrella/EternalJukebox/issues/31))

### Description
When I try to search for songs, it doesn't load for me. I only see this blue loading line but when it reaches the end, I still only see the recommendations, please help me

**Comment by AlpAltunkaya** [comment]
I have the exact same problem and have been checking the website multiple times in a day for a week. Would I be able to use the jukebox if I hosted it on my server? Any ideas?

**Comment by Piercekathrine** [comment]
Yeah. Even my search songs are not loading at all. I have checked it for the Nth time... How can I fix this pls respond..

**Comment by chiarastrq** [comment]
Hey, sorry, but I can't help you with this, because it still
doesn't work for me as well :/

**Comment by IanSteg** [comment]
@AlpAltunkaya Yes, if you self host, it should work better.
You can also try this site: https://jukebox.stegner.games/

If it still doesn't work, try these steps:

- Open https://music.youtube.com/
- Search for your song
- Find a result that matches your Spotify track
- Right Click > Share
- Then, in your Eternal Jukebox page for the song, click 'Tune', and paste that url into the 'Audio URL' field
- Press 'enter', and wait to be redirected
- Be patient!

You're also able to upload a song you have locally, if that will work better:
- In your Eternal Jukebox page for the song, click 'Tune', and then click upload
- Find your file, and select it
- Wait for it to upload
- Press 'enter', and wait to be redirected
- Be patient!
- (It's recommended to bookmark this URL, to prevent reuploading the audio again and again, and to save bandwidth on my end)

This isn't guaranteed to work all the time

**Comment by AlpAltunkaya** [comment]
@IanSteg thank you so much for rehosting the jukebox, all is working perfectly! The steps you mentioned for importing a song weren't even working with the original website anymore, that's how buggy it got. 

I will be definitely saving the URL if I upload a file of my own, your wish is duly noted!
